types/unbzip2-stream/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

```suggestion
        "forceConsistentCasingInFileNames": true
```
why you want 'esModuleInterop' here?
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

---

@peterblazejewicz I added it because the linting errored without it. If I remove `"esModuleInterop": true` I get:
```
$ npm test unbzip2-stream

> definitely-typed@0.0.3 test
> dtslint types "unbzip2-stream"

dtslint@0.0.110
Error: /home/me/projects/DefinitelyTyped/types/unbzip2-stream/unbzip2-stream-tests.ts:1:8
ERROR: 1:8  expect  TypeScript@4.7 compile error: 
Module '"stream"' can only be default-imported using the 'esModuleInterop' flag
ERROR: 2:8  expect  TypeScript@4.7 compile error: 
Module '"/home/me/projects/DefinitelyTyped/types/unbzip2-stream/index"' can only be default-imported using the 'esModuleInterop' flag

    at testTypesVersion (/home/me/projects/DefinitelyTyped/node_modules/@definitelytyped/dtslint/dist/index.js:194:15)
    at async runTests (/home/me/projects/DefinitelyTyped/node_modules/@definitelytyped/dtslint/dist/index.js:170:13)
    at async main (/home/me/projects/DefinitelyTyped/node_modules/@definitelytyped/dtslint/dist/index.js:79:9)
```

Is there something different I should do?

---

@peterblazejewicz Ah! I think I've worked out that it was because I was doing the imports in the test wrong. I hope I have fixed it in my new commit.